agents/plugins/mk_oracle

```diff
@@ -1935,7 +1950,7 @@ do_custom_sqls () {
         fi
 
         # set custom credentials from section
-        db_connect=$(mk_ora_db_connect "$sid")
+        db_connect=$(mk_ora_db_connect "$MK_SID")
```

Comment:
This is a fix which has to be picked to 1.6.0 branch. Please do not mix bug fix and feature.